.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+_data/ @TomerFi @bennypowers
+_includes/ @bennypowers
+assets/ @bennypowers
+feed/ @bennypowers
+/*.* @bennypowers @TomerFi
+.github/ @TomerFi
+*.njk @bennypowers @TomerFi
```

Comment:
you'll get pinged on layout and templates stuff

Reply:
yes, are they changing frequently?
I figured, once we get rolling, most changes would be in the posts folder no?

if I remove myself from this?
remove you as well or are you ok for being notified for *.njk files?

Reply:
you know what... let's keep it like this for now.
I'll remove myself if it get out of hand.